src/System.Data.Odbc/src/Common/System/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -0,0 +1,258 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Globalization;
+
+namespace System.Data.Common
+{
+    internal static class DbConnectionStringDefaults
```

Comment:
The same class is present in System.Data.SqlClient 
https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/src/System/Data/Common/DbConnectionStringCommon.cs#L227

It would be good to consolidate this class an other duplicate classes from SqlClient.

If you have a file in System/Data/Common folder, you should check if System.Data.SqlClient or System.Data.Common have those classes as well. 
If they do, it would be a good idea to extract the common pieces and move them to the CoreFx Common Folder and reference the common partial class from the Common folder in all the assemblies.